llvm/include/llvm/ADT/APInt.h

```diff
@@ -1626,7 +1626,8 @@ class [[nodiscard]] APInt {
   /// SmallString. If Radix > 10, UpperCase determine the case of letter
   /// digits.
   void toString(SmallVectorImpl<char> &Str, unsigned Radix, bool Signed,
-                bool formatAsCLiteral = false, bool UpperCase = true) const;
+                bool formatAsCLiteral = false, bool UpperCase = true,
+                bool insertSeparators = false) const;
```

Comment:
```suggestion
                bool InsertSeparators = false) const;
```
This matches our usual coding style and is the prevailing style in the function signature (a follow-up NFC change could rename `formatAsCLiteral`).